Add support for list of bigrams (#23)
words/dice-coefficient: readme.md

diceCoefficient('abc', 'xyz') // => 0
diceCoefficient('night', 'nacht') // => 0.25
diceCoefficient('night', 'nacht') === dice('NiGhT', 'NACHT') // => true

// bigrams may also be passed as input arguments for improved efficiency
// when analyzing the same strings repeatedly, for example, when
// comparing the text of each file in a directory with the text of
// each file in another directory.

import {bigram} from 'n-gram'

const bigramifiedString1 = bigram('abc') // ['ab', 'bc']
const bigramifiedString2 = bigram('xyz') // ['xy', 'yz']

> diceCoefficient(bigramifiedString1, bigramifiedString2) // => 0

Reviewer: I don‘t think this needs to be in the Use section. but it should probably be in the API section, that arrays of strings are allowed, and a note that they should be bigrams?

Author: Moved the explanation to another code section, if that works.

diceCoefficient('night', 'nacht') === dice('NiGhT', 'NACHT') // => true
```

> Bigrams (arrays of 2-character-length strings) may also be passed as input arguments for better efficiency when processing the same strings repeatedly.

Reviewer: ```suggestion
Instead of strings you can also pass lists of bigrams.
This can improve performance when processing the same strings repeatedly.
```

```js
diceCoefficient(['ab', 'bc'], ['xy', 'yz']) // => 0
diceCoefficient(['ab', 'bc'], ['ab', 'bc']) // => 1
diceCoefficient(['ab', 'bc'], ['AB', 'BC']) // => 1
```

> See [n-gram](https://github.com/words/n-gram) for a helpful utility function for generating bigrams.

Reviewer: ```suggestion
See [`n-gram`](https://github.com/words/n-gram) for generating bigrams.
```

```js

> import {bigram} from 'n-gram'
> const bigramifiedString1 = bigram('abc') // => ['ab', 'bc']
> const bigramifiedString2 = bigram('xyz') // => ['xy', 'yz']
> 
> diceCoefficient(bigramifiedString1, bigramifiedString2) // => 0

Reviewer: ```suggestion
import {bigram} from 'n-gram'

const abc = bigram('abc') // => ['ab', 'bc']
const xyz = bigram('xyz') // => ['xy', 'yz']

diceCoefficient(abc, xyz) // => 0
```
